liquidctl/driver/kraken3.py

```diff
@@ -806,9 +814,16 @@ def check_unsupported_fw_version():
             self._write([0x30, 0x02, 0x01, self.brightness, 0x0, 0x0, 0x1, int(value_int / 90)])
             return
         elif mode == "static":
-            check_unsupported_fw_version()
-            data = self._prepare_static_file(value, self.orientation)
-            self._send_data(data, [0x02, 0x0, 0x0, 0x0] + list(len(data).to_bytes(4, "little")))
+            if is_fw_version2():
+                data = self._prepare_static_file_rgb16(value, self.orientation)
+                self._send_data_fw2(data, [0x06, 0x0, 0x0, 0x0] + list(len(data).to_bytes(4, "little")))
+                # sending it twice is only required once after initialization
+                # the same behaviour is observed in manufacturer at init software
```

Comment:
Nit: `at software init`?

Reply:
More firmware init, anytime usb is disconnected or device rebooted